src/lib/operators/aggregate_hash.cpp

```diff
@@ -316,15 +348,74 @@ void AggregateHash::_aggregate() {
                     keys_per_chunk[chunk_id][chunk_offset][group_column_index] = 0u;
                   }
                 } else {
-                  auto inserted = id_map.try_emplace(position.value(), id_counter);
+                  auto id = AggregateKeyEntry{0} - 1;                          // using max value for "unset"
```

Comment:
Not obvious that you are causing "a flip"  (don't know the term) here. Extend comment?

Reply:
See previous comment. Better?